catalog.html

```diff
@@ -270,20 +276,23 @@ <h2 class="subtitle-searchhotel">
       <h2 class="visually-hidden">Мы в социальных сетях</h2>
       <ul>
         <li>
-          <a class="button" href="https://twitter.com/htmlacademy_ru">
+          <a class="button-social" href="https://twitter.com/htmlacademy_ru">
             <span class="visually-hidden">Твиттер</span>
+            <img src="images/social/twitter.svg" alt="">
           </a>
```

Comment:
А давайте сразу облегчим верстку
```suggestion
          <a class="button-social" href="https://twitter.com/htmlacademy_ru">
            <img src="images/social/twitter.svg" alt="Твиттер">
          </a>
```